hazelcast-jet-core/src/main/java/com/hazelcast/jet/impl/JetService.java

```diff
@@ -104,6 +104,7 @@ public void init(NodeEngine engine, Properties hzProperties) {
         jobRepository = new JobRepository(jetInstance);
         jobExecutionService = new JobExecutionService(nodeEngine, taskletExecutionService, jobRepository);
         jobCoordinationService = createJobCoordinationService();
+        nodeEngine.getMetricsRegistry().registerDynamicMetricsProvider(jobCoordinationService);
```

Comment:
can we group the registrations together(with jobExecutionService)

Reply:
No longer needed, `JobCoordinationService` will not be a dynamic metrics provider.